README.md

```diff
@@ -20,8 +20,8 @@
 
 ## Introduction
 
-**nfcore/ampliseq** is a bioinformatics analysis pipeline used for amplicon sequencing data, supporting 16S, ITS and 18S data. Supported is paired-end Illumina or single-end Illumina, PacBio and IonTorrent data.
-
+**nfcore/ampliseq** is a bioinformatics analysis pipeline used for amplicon sequencing, supporting 16S, ITS and 18S data. Supported is paired-end Illumina or single-end Illumina, PacBio and IonTorrent data. Default is the analysis of 16S rRNA gene amplicons sequenced paired-end with Illumina.
```

Comment:
I made a mistake when reviewing this line, so now the old and new are both present.

Reply:
Or maybe they're not? I'm confused.

Reply:
Indeed, doubled the line: https://github.com/d4straub/ampliseq/tree/dev, I'll fix it.